docs/docs/features/hardware-transcoding.md

```diff
@@ -42,13 +42,21 @@ As this is a new feature, it is still experimental and may not work on all syste
   - Additionally, if the server specifically has an 11th gen CPU and is running kernel 5.15 (shipped with Ubuntu 22.04 LTS), then you will need to upgrade this kernel (from [Jellyfin docs][jellyfin-kernel-bug])
 
 ## Setup
-
+#### Initial Setup
 1. If you do not already have it, download the latest [`hwaccel.yml`][hw-file] file and ensure it's in the same folder as the `docker-compose.yml`.
 2. Uncomment the lines that apply to your system and desired usage.
 3. In the `docker-compose.yml` under `immich-microservices`, uncomment the lines relating to the `hwaccel.yml` file.
 4. Redeploy the `immich-microservices` container with these updated settings.
 5. In the Admin page under `FFmpeg settings`, change the hardware acceleration setting to the appropriate option and save.
 
+#### Imagegenius/Immich docker Community Appstore Setup
+##### NVENC - NVIDIA GPUs
```

Comment:
In addition to this section, it would be good to mention that for other backends, the relevant contents of the `hwaccel.yml` file need to be inlined into the `immich-microservices` service directly. This is because Unraid doesn't support deploying with multiple Compose files.